src/views/developers/l10n.json

```diff
@@ -9,12 +10,19 @@
+  "developers.scratchBlocksIntroBlocklyLinkText": "Blockly technology",
+  "developers.scratchBlocksIntroGitHubLinkText": "here",
+  "developers.scratchBlocksIntro": "Scratch Blocks is a new development project for the next generation of graphical programming blocks, based on a collaboration between Google and MIT’s Scratch Team — building on Google’s {blocklyLink} and informed by the Scratch Team’s expertise in developing creative learning tools for young people. Scratch Blocks will provide a framework for building programming blocks in both vertical (text-based) and horizontal (icon-based) formats. You can access the code (currently as a developer-preview) and documentation {githubLink}.",
   "developers.scratchBlocksBody": "This first release includes code for Scratch’s Horizontal Grammar. Looking ahead, we plan to release additional code including but not limited to the Vertical Grammar (currently used by Scratch), a new Rendering Engine to support sprites and graphic effects, and a new Audio Engine to support creation with sound and music.",
   "developers.wwwTitle": "Scratch WWW",
-  "developers.wwwIntro": "Scratch-www is a standalone web client for the Scratch Community, built using React and Redux. Access the code and documentation through Github <a href=\"https://github.com/LLK/scratch-www\">here</a>.",
-  "developers.principlesIntro": "We created Scratch to empower young people to think creatively, reason systematically, and work collaboratively. We are guided by a set of <b>Learning Principles</b> and <b>Design Principles</b> that we hope you will follow as you develop new tools and technologies with Scratch Blocks.",
-  "developers.jrBody": "ScratchJr is an introductory programming language that enables young children (ages 5-7) to create their own interactive stories and games. For more information, visit the <a href=\"https://www.scratchjr.org/\">ScratchJr website</a> or access the code and documentation <a href=\"https://github.com/LLK/scratchjr\">on GitHub</a>.",
+  "developers.wwwIntroLinkText": "here",
+  "developers.wwwIntro": "Scratch-www is a standalone web client for the Scratch Community, built using React and Redux. Access the code and documentation through Github {wwwIntroLink}.",
+  "developers.principlesIntroLearningPrinciples": "Learning Principles",
+  "developers.principlesIntroDesignPrinciples": "Design Principles",
+  "developers.principlesIntro": "We created Scratch to empower young people to think creatively, reason systematically, and work collaboratively. We are guided by a set of {learningPrinciples} and {designPrinciples} that we hope you will follow as you develop new tools and technologies with Scratch Blocks.",
+  "developers.jrBodyWebsiteLinkText": "ScratchJr website",
+  "developers.jrBodyGitHubLinkText": "on GitHub",
```

Comment:
I'm not sure why we made 'on' part of the link. Let's move it to the original string, and just have `Github` as the link text, and that won't need to be translated.